raft/raft.go

```diff
@@ -499,9 +487,6 @@ func (r *Raft) sendAppend(to uint64) {
 // are undesirable when we're sending multiple messages in a batch).
 func (r *Raft) maybeSendAppend(to uint64, sendIfEmpty bool) bool {
 	pr := r.getProgress(to)
-	if pr.IsPaused() {
```

Comment:
removing `SnapshotStatus` will make generate multiple snapshots every time when there is a snapshot is on the flight.

Reply:
Yes, you are right, `Snapshot` will be like `AppendLog` except contains a big `state machine` not `log entries`, should we keep flow control for snapshot?

Reply:
alright, seems no correctness issue. @hicqu Is tikv snapshot mechanism based on the flow control of snapshot?